src/modules/Library/features/Popups/AddToBuilderPopup/AddToBuilderPopup.utils.tsx

```diff
@@ -265,8 +267,8 @@ export const getSteps = ({
 };
 
 export const getArrayFromApplets = (applets: FullApplet[]) =>
-  applets.reduce((acc: Applet[], { id, type, displayName, image }) => {
-    if (type === DashboardAppletType.Applet) {
+  applets.reduce((acc: Applet[], { id, type, displayName, image, role }) => {
+    if (type === DashboardAppletType.Applet && (role === Roles.Owner || role === Roles.Manager)) {
```

Comment:
same here

Reply:
as I understand in applets list there is only one main role which is a string
![image](https://github.com/ChildMindInstitute/mindlogger-admin/assets/116339462/32842a39-61f5-484c-a8cb-60da85e4746c)